pkgs/applications/networking/soju/default.nix

```diff
@@ -18,13 +18,6 @@ buildGoModule rec {
 
   vendorHash = "sha256-iT/QMm6RM6kvw69Az+aLTtBuaCX7ELAiYlj5wXAtBd4=";
 
-  subPackages = [
-    "cmd/soju"
-    "cmd/sojuctl"
-    "contrib/migrate-db"
-    "contrib/znc-import"
-  ];
-
```

Comment:
is this superfluous?

Reply:
Please add `cmd/sojudb` and `contrib/migrate-logs` here instead of deleting this attribute.